### PR TITLE
"Build and Load" Documentation

### PR DESCRIPTION
**Pull request by btmonier**
## Description

This PR is for the first draft of the "Build and Load" documentation, which details the initial steps of the PHGv2 pipeline.

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
Initial commit of 'build_and_load.md' document.
```

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+* Basic syntax is:
+  ```shell
+  phg [<options>] <command> [<args>]...
+  ```
+
+## Requirements
+PHGv2 requires basic software components: a Unix-style operating 
+system and Java version 17 or higher. PHGv2 
+also relies on external programs for alignment and storage, including 
+[AnchorWave](https://github.com/baoxingsong/AnchorWave) and
```
Comment (lynnjo) [inline docs/installation.md:43]: Should you note here that anchorwave does not support Windows?  It only runs on linux/mac (but not currently ARM mac)

File: docs/installation.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+presume you will run the program directly. Obtain the `.tar` file 
+manually from the provided link or use the following `curl` and `awk` 
+commands to retrieve the latest release:
+
+```shell
+curl -s https://api.github.com/repos/maize-genetics/phg_v2/releases/latest \
+| awk -F': ' '/browser_download_url/ && /\.tar/ {gsub(/"/, "", $(NF)); system("curl -LO " $(NF))}'
+```
+
+Once downloaded, simple untar the release using:
```
Comment (lynnjo) [inline docs/installation.md:68]: should this be "simply" vs "simple"?

File: docs/build_and_load.md [hunk 3]
```diff
@@ -0,0 +1,766 @@
...
+> [---------Merged overlap-----------]
+>  ```
+  
+* `-o` - Name for the output BED file.
+
+In the above example, I am using test data from the 
+[PHGv2 GitHub repository](https://github.com/maize-genetics/phg_v2/tree/main/data/test/smallseq).
+After the command is finished, we have produced a BED file (which
+in my case, is located in a subdirectory labelled `output`). This BED 
+file contains 4 columns of information:
```
Comment (lynnjo) [inline docs/build_and_load.md:260]: Looking at the code, there are actually 6 columns returned.  contig, start, end, name, score and strand.  The "name" is the gene or cds name.  If regions overlapped and were merged, the name is a comma separated string of all the genes/cds included in this bed region.  THe score appears to always be 0 - not sure what that is supposed to represent (Zack would know) nor am I sure why strand is included as I thought bed files were always on the forward strand.

**Review by zrm22 — APPROVED** [review]

**Review by lynnjo — APPROVED** [review]

**Review by tcasstevens — APPROVED** [review]

File: docs/installation.md (hunk 1 above)
Comment (btmonier) [reply docs/installation.md:43]: Updated to add minor requirement info for AnchorWave

**Review by pjbradbury — APPROVED** [review]

**Pull request merged**